roles/gitops/rendering-apps-files/templates/nexus/templates/prometheusrule.yml.j2

```
@@ -55,10 +57,11 @@ spec:
       for: 1m
       labels:
         severity: critical
+
     - alert: Nexus PVC out of disk space
       annotations:
-        message: PVC {{"{{"}} $labels.persistentvolumeclaim {{"}}"}} in namespace {{ dsc.nexus.namespace }} is full (0% left).
-        summary: Nexus PVC out of disk space in namespace {{ dsc.nexus.namespace }}
+        {% raw %}message: {{ printf "PVC {{ $labels.persistentvolumeclaim }} in namespace {% endraw %}{{ dsc.nexus.namespace }}{% raw %} is full (0%% left)."{% endraw %} {{"}}"}}
```

```suggestion
        {% raw %}message: {{ printf "PVC {{ $labels.persistentvolumeclaim }} in namespace {% endraw %}{{ dsc.nexus.namespace }}{% raw %} is full (0% left)."{% endraw %} {{"}}"}}
```

Erreur de frappe sur "(0% left)".